src/VisualStudio/Core/Def/Implementation/Diagnostics/VisualStudioWorkspaceDiagnosticAnalyzerProviderService.cs

```diff
@@ -124,7 +124,8 @@ internal static ImmutableArray<HostDiagnosticAnalyzerPackage> GetHostAnalyzerPac
 
                 var packages = builder.ToImmutable();
 
-                EnsureMandatoryAnalyzers(packages);
+                // TODO: This crashes my cloud client, so commenting out for now.
+                // EnsureMandatoryAnalyzers(packages);
```

Comment:
@dibarbet Do you have to comment this out to get things working for you? @mavasani Any ideas why this wouldn't work in connected scenarios?

Reply:
VSIX Analyzers are detected by Analyzer assets added in the VSIX manifest. Mandatory analyzers (for C# and VB compiler diagnostics) are defined in MS.CA.CSharp.dll and MS.CA.VisualBasic.dll and specified as assets here: https://github.com/dotnet/roslyn/blob/a88aa571636d1347be750f88822edaa6a4f3f445/src/VisualStudio/Setup/source.extension.vsixmanifest#L57-L58

Reply:
This is a problem with `master-vs-deps` as well, so it's not new in this PR.

Reply:
@tmat @jasonmalinowski Would you expect these mandatory analyzers to be present on a client machine? Or is them not being there a good thing?

Reply:
I do correctly get diagnostics in the error list without these loaded (as expected, since the server is doing the work)...

Reply:
We use the VS `ExtensionManagerService` to `GetEnabledExtensions`, of which C#/VB are not.